src/index.html

```diff
@@ -0,0 +1,329 @@
+              fill="#1E1B1B"
+            />
+          </svg>
+        </a>
+        <ul class="nav__menu">
+          <li>
+            <a class="nav__link" href=""> Home </a>
+          </li>
+          <li>
+            <a class="nav__link" href="product.html"> Product </a>
+          </li>
+          <li>
+            <a class="nav__link" href=""> Pricing </a>
+          </li>
+          <li>
+            <a class="nav__link" href=""> About us </a>
+          </li>
+          <li>
+            <a class="nav__link" href=""> Blog </a>
+          </li>
+          <li>
+            <a class="nav__link" href=""> Contact </a>
+          </li>
+          <li>
+            <a class="button button--primary ml-5" href="">Free trial</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="hero relative">
```

Comment:
`relative` global modifiers are not allowed in BEM. `tailwind` classes are not allowed